packages/frontend/src/components/EditorLayout/index.tsx

```diff
@@ -118,27 +108,20 @@ export default function EditorLayout(): React.ReactElement {
             colorScheme="secondary"
             target="_blank"
             variant="link"
-            mr={6}
+            pr={4}
             _hover={{ textDecoration: 'underline' }}
           >
             Guide
           </Button>
 
           <Box pr={1}>
-            <MuiButton
-              variant="contained"
-              size="small"
-              onClick={() => onFlowStatusUpdate(!flow.active)}
-              data-test={
-                flow?.active ? 'unpublish-flow-button' : 'publish-flow-button'
-              }
-            >
-              {flow?.active
-                ? formatMessage('flowEditor.unpublish')
-                : formatMessage('flowEditor.publish')}
-            </MuiButton>
+            <Button size="md" onClick={() => onFlowStatusUpdate(!flow.active)}>
+              <Text textStyle="subhead-1">
+                {flow?.active ? 'Unpublish' : 'Publish'}
```

Comment:
We removing i18n here?

Reply:
We're indeed progressively removing i18n
"ian says we not supporting multi language" - @m0nggh, 2023